src/Twig/StorageExtension.php

```diff
@@ -24,8 +26,14 @@ public function getFilters(): array
         ];
     }
 
-    public function formatUrl(string $path): string
+    public function formatUrl(string $path, AssetsHelper $assetsHelper): string
     {
-        return $this->awsS3PublicUrlResolver->resolve($path);
+        // dump( preg_match(self::AUTHORIZED_SCHEMA_REGEX, $path) );
+
+        if (preg_match(self::AUTHORIZED_SCHEMA_REGEX, $path)) {
+            $assetsPath = $assetsHelper->getUrl('/images/default_speaker.svg');
```

Comment:
What is this doing? Should be return instead of assign shouldn't it?

Reply:
Also, `path` should be either the relative path to the image (so aws) or the absolute path (in case of the default image for instance), so it should just return path